### PR TITLE
Changed evidences to have it's own ids

### PR DESCRIPTION
**Pull request by BJSS-RicardoTavares**

### CODE REVIEW

File: src/modules/shared/pages/innovation/sections/section-info.component.html [hunk 1]
```diff
@@ -83,71 +80,67 @@ <h2 class="nhsuk-heading-m">{{ isInnovatorType ? 'Your' : 'Submitted' }} answers
...
+            </dd>
+          </div>
         </dl>
+
       </ng-container>
 
       <div *ngIf="isInnovatorType">
-        <button *ngIf="section.showSubmitButton" (click)="onSubmitSection()" class="nhsuk-button nhsuk-u-margin-right-3"> Confirm section answers </button>
-        <button *ngIf="section.showSubmitUpdatesButton" routerLink="/innovator/innovations/{{ innovation.id }}/record/sections/{{ section.id }}/confirm-update" class="nhsuk-button nhsuk-u-margin-right-3"> Submit updates </button>
+        <button *ngIf="section.showSubmitButton" (click)="onSubmitSection()" class="nhsuk-butto nhsuk-u-margin-top-5">Confirm section answers</button>
```
Comment (diogo-costa-bjss) [inline src/modules/shared/pages/innovation/sections/section-info.component.html:117]: Typo on class `nhsuk-button`

File: src/modules/stores/innovation/innovation.service.ts [hunk 2]
```diff
@@ -57,87 +37,61 @@ export class InnovationService {
...
     const url = new UrlModel(this.API_INNOVATIONS_URL).addPath('v1/:innovationId/sections/:sectionKey/submit').setPathParams({ userId: this.authenticationStore.getUserId(), innovationId, sectionKey });
-    return this.http.patch<any>(url.buildUrl(), { sectionKey }).pipe(
-      take(1),
-      map(response => response)
-    );
+    return this.http.patch<any>(url.buildUrl(), { sectionKey }).pipe(take(1), map(response => response));
   }
 
 
   getSectionEvidenceInfo(innovationId: string, evidenceOffset: string): Observable<GetInnovationEvidenceDTO> {
```
Comment (diogo-costa-bjss) [inline src/modules/stores/innovation/innovation.service.ts:64]: Change `evidenceOffset` param to `evidenceId` to maintain consistency

Comment (BJSS-RicardoTavares) [reply src/modules/stores/innovation/innovation.service.ts]: Solved. Thanks.

File: src/modules/shared/pages/innovation/sections/section-info.component.html (hunk 1 above)
Comment (BJSS-RicardoTavares) [reply src/modules/shared/pages/innovation/sections/section-info.component.html]: Solved. Thanks.

**Review by diogo-costa-bjss — APPROVED** [review]

**Pull request merged**